ethereum_history_api/oracles/src/noir/oracles/accountOracle/encode.ts

```diff
@@ -10,6 +10,10 @@ const MAX_ACCOUNT_STATE_LENGTH = 134;
 export const STATE_PROOF_LENGTH = PROOF_ONE_LEVEL_LENGTH * MAX_STATE_PROOF_LEVELS;
 const MAX_STORAGE_PROOF_LEVELS = 7;
 export const STORAGE_PROOF_LENGTH = PROOF_ONE_LEVEL_LENGTH * MAX_STORAGE_PROOF_LEVELS;
+export const MAX_RECEIPT_TREE_DEPTH = 6;
+const MAX_RECEIPT_TRIE_NODE_LENGTH = 515;
+export const MAX_RECEIPT_PROOF_LENGTH = MAX_RECEIPT_TRIE_NODE_LENGTH * MAX_RECEIPT_TREE_DEPTH;
+export const MAX_RECEIPT_RLP_LENGTH = 512; // TODO: Put the real value here
```

Comment:
Let's reuse MAX_RECEIPT_TRIE_NODE_LENGTH for now
```suggestion
export const MAX_RECEIPT_RLP_LENGTH = MAX_RECEIPT_TRIE_NODE_LENGTH;
```